### PR TITLE
RD-2593 Hide "Edit a copy in Composer" button

### PR DESCRIPTION
**Pull request by Gelio**
This PR hides the "Edit a copy in Composer" button by default in the Blueprint Action Buttons widget. It can be brought back by changing the configuration.

## Video

https://user-images.githubusercontent.com/889383/123640312-536b6680-d821-11eb-970f-c10363564dce.mp4

## System tests

https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/857/pipeline

Queued 2021-06-28 14:59 CEST

### CODE REVIEW

**Review by kubama — COMMENTED** [review]
General comment to all `...EditACopyInComposerButton` identifiers - I think including articles like 'a' or 'the' in identifier names is very uncommon. My first impression when reading it was that there is copy 'A', which it refers to (like if there also was copy 'B', 'C' and so on). How about changing it?

**Comment by Gelio** [comment]
Sure, let's go with just `EditCopyInComposer` when it's camelcased/pascalcased :slightly_smiling_face:

**Review by kubama — DISMISSED** [review]

**Comment by Gelio** [comment]
I have encountered a (most likely) flaky failure in `filters_spec` and dropdowns. I have reported RD-2664, as I have seen it a few times already.

I don't see it as being in any way related to the Blueprint Action Buttons widget or other changes in this file, so I will assume that the system tests passed :smile:

File: widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ComponentProps } from 'react';
 interface BlueprintActionButtonsProps {
     blueprintId?: string;
     toolbox: Stage.Types.Toolbox;
+    showEditCopyInComposerButton: boolean;
```
Comment (stepek) [inline widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx:6]: minor:
name of this props is to long imho, don't have idea how to reduce the length of that

**Review by stepek — DISMISSED** [review]
Code LGTM!

File: widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx (hunk 1 above)
Comment (Gelio) [reply widgets/blueprintActionButtons/src/BlueprintActionButtons.tsx:6]: Yup, I understand, and I also had a doubt writing such a long name, but I also did not come up with a better one that would make it as descriptive :slightly_smiling_face: I hope it's ok if it stays

**Comment by Gelio** [comment]
I have rebased the branch on `6.1.0` as #1472 was merged. No code changes were introduced

**Review by kubama — APPROVED** [review]

**Pull request merged**